examples/02_06_specific_words_with_enrichment.ipynb

```diff
@@ -0,0 +1,826 @@
+    "tokens_list = preprocess(texts)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Najprej si izpišimo vse besede/izraze, ki smo jih dobili po preprocesiranju četrtega dokumenta. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/markdown": [
+       "trenuten, obdavčitev, stanovanje, 120m2, hiša, 250m2, površina, presegati, cenzus, celoten, cena, obračun, ddv, stopnja, posledičen, stanovanje, površina, 121m2, več, dvakrat, bolj, obdavčen, stanovanje, površina, 119m2, absurden, predlagati, obdavčitev, spremeniti, površina, 120m2, stanovanje, 250m2, hiša, vedno, obdavčen, površina, presegati, cenzus, visok, obdavčen"
+      ],
+      "text/plain": [
+       "<IPython.core.display.Markdown object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "display(Markdown(\", \".join(tokens_list[4])))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "V naslednjem koraku izračunamo značilke z metodo imenovano vreča based."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "joined_texts = [' '.join(tokens) for tokens in tokens_list]\n",
+    "vectorizer = TfidfVectorizer()\n",
+    "X = vectorizer.fit_transform(joined_texts)\n",
+    "words = vectorizer.get_feature_names()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/markdown": [
+       "Število besed v slovarju: 2270"
+      ],
+      "text/plain": [
+       "<IPython.core.display.Markdown object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "display(Markdown(f\"Število besed v slovarju: {len(words)}\"))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Ko imamo značilke lahko na podlagi teh najdemo značilne besede z metodo obogatitev besed. V prvem koraku bomo napisali funkcijo, ki izračuna p-vrednosti in FDR (popravljeno p-vrednost) za vsako besedo iz množice vseh besed za podani dokument. Funkcija vrne zgolj besede, ki imajo p-vrednosti manjše od 1. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def specific_words(document_index):\n",
+    "    p_values = hypergeom_p_values(X, X[document_index])\n",
+    "    fdr = FDR(p_values)\n",
+    "    df = pd.DataFrame(\n",
+    "        {\"Besede\": words, \"P-Vrednost\": p_values, \"FDR\": fdr}\n",
+    "    ).sort_values(\"P-Vrednost\", axis=0)\n",
+    "    return df[df[\"P-Vrednost\"] < 1]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Pokažimo besedilo in začilne besede za dva dokumenta."
```

Comment:
začilne --> značilne

Reply:
Thank you for your corrections. I should stop to relay on automatic word corrections.